ui/src/onboarding/components/AdminStep.tsx

```diff
@@ -221,7 +225,8 @@ class AdminStep extends PureComponent<Props, State> {
     const {confirmPassword} = this.state
     const password = e.target.value
     const isPassMismatched = confirmPassword && password !== confirmPassword
-    this.setState({password, isPassMismatched})
+    const isPassTooShort = password && password.length < 8
```

Comment:
what's the intent of checking for `password` here? is it to guard against empty strings triggering validation?

i noticed that there aren't any test cases for "" in the password e2e tests - are these things related or did i miss those?

Reply:
that's correct.  i don't want to it to show up when the password is empty.